hack/build.sh

```diff
@@ -108,23 +108,9 @@ yq eval -i '
 docker buildx build --push -f ./bundle/Dockerfile --platform "linux/amd64,linux/ppc64le,linux/s390x"  -t "${REGISTRY}/${NAMESPACE}/quay-operator-bundle:${TAG}" ./bundle
 digest "${REGISTRY}/${NAMESPACE}/quay-operator-bundle:${TAG}" BUNDLE_DIGEST
 
-AMD64_DIGEST=$(docker manifest inspect --verbose ${REGISTRY}/${NAMESPACE}/quay-operator-bundle:${TAG} | \
-    jq -r 'if type=="object"
-        then .Descriptor.digest
-        else .[] | select(.Descriptor.platform.architecture=="amd64" and .Descriptor.platform.os=="linux") | .Descriptor.digest
-	end')
-
-POWER_DIGEST=$(docker manifest inspect --verbose ${REGISTRY}/${NAMESPACE}/quay-operator-bundle:${TAG} | \
-    jq -r 'if type=="object"
-        then .Descriptor.digest
-        else .[] | select(.Descriptor.platform.architecture=="ppc64le" and .Descriptor.platform.os=="linux") | .Descriptor.digest
-        end')
-
-Z_DIGEST=$(docker manifest inspect --verbose ${REGISTRY}/${NAMESPACE}/quay-operator-bundle:${TAG} | \
-    jq -r 'if type=="object"
-        then .Descriptor.digest
-        else .[] | select(.Descriptor.platform.architecture=="s390x" and .Descriptor.platform.os=="linux") | .Descriptor.digest
-        end')
+AMD64_DIGEST=$(skopeo inspect --raw  docker://${REGISTRY}/${NAMESPACE}/quay-operator-bundle:3.9-unstable | jq -r '.manifests[0].digest')
```

Comment:
Please replace `3.9-unstable` with `${TAG}`

Reply:
Also you shouldn't rely on manifests order. It's better to use
```
jq -r '.manifests[] | select(.platform.architecture == "amd64" and .platform.os == "linux").digest'
```

Reply:
ok